plugin.py

```diff
@@ -17,6 +21,25 @@ class LspTailwindcssPlugin(NpmClientHandler):
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        _window: sublime.Window,
+        _initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        _configuration: Optional[ClientConfig] = None
```

Comment:
Prefixing arguments with an underscore is pretty atypical convention if you ask me. :)
I understand that you want to indicate that those are unused but it doesn't even silence the linter so is it worth it?

Reply:
Not worth it. :)

I use this when writing typescript... in typescript it silents the error